lib/options.json

```diff
@@ -1076,6 +1071,64 @@
       "type": "string",
       "minLength": 1
     },
+    "WatchOptionsChokidar": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Configure advanced options for watching. See the chokidar documentation for the possible options.",
+      "link": "https://github.com/paulmillr/chokidar#api",
+      "properties": {
+        "persistent": {
+          "type": "boolean"
+        },
+        "ignored": {
+          "type": "string"
+        },
+        "ignoreInitial": {
+          "type": "boolean"
+        },
+        "followSymlinks": {
+          "type": "boolean"
+        },
+        "cwd": {
+          "type": "string"
+        },
+        "disableGlobbing": {
+          "type": "boolean"
+        },
+        "usePolling": {
+          "type": "boolean"
+        },
+        "interval": {
+          "type": "number"
+        },
+        "binaryInterval": {
+          "type": "number"
+        },
+        "alwaysStat": {
+          "type": "boolean"
+        },
+        "depth": {
+          "type": "number"
+        },
+        "awaitWriteFinish": {
+          "type": "object",
+          "additionalProperties": true
+        },
+        "ignorePermissionErrors": {
+          "type": "boolean"
+        },
+        "atomic": {
+          "anyOf": [
+            {
+              "type": "boolean"
+            },
+            {
+              "type": "number"
+            }
+          ]
+        }
+      }
+    },
```

Comment:
hm, so chokidar doesn't validate own options?

Reply:
@alexander-akait No, it doesn't.


https://user-images.githubusercontent.com/46647141/166394402-7898c98b-c3b7-4181-bc8c-4d0ec2b907d6.mov

Reply:
hmm, I think we allow some extra options for watching, but `chokidar` will only understand its API options.

https://github.com/webpack/webpack-dev-server/blob/c2fee3b585307017c66b322afd78220e6b6c835a/lib/Server.js#L766-L772

https://github.com/webpack/webpack-dev-server/blob/c2fee3b585307017c66b322afd78220e6b6c835a/lib/Server.js#L805

Reply:
Yeah, ideally chokidar should have its own validation. Maybe we can add an option in scema-utils `additionalProperties: "warn"` which will warn if you use another property instead of throwing an error and exiting the process.

Reply:
Sounds better, we will still needed update them when new options were added, but it happens rarely